packages/blocks/src/page-block/edgeless/components/toolbar/note/note-memu-config.ts

```diff
@@ -0,0 +1,123 @@
+import {
+  BulletedListIcon,
+  CodeBlockIcon,
+  H1Icon,
+  H2Icon,
+  H3Icon,
+  H4Icon,
+  H5Icon,
+  H6Icon,
+  NumberedListIcon,
+  QuoteIcon,
+  TextIcon,
+  TodoIcon,
+} from '@blocksuite/global/config';
+import { type TemplateResult } from 'lit';
+
+import {
+  type NoteChildrenFlavour,
+  type NoteChildrenType,
+} from '../../../../../__internal__/index.js';
+
+export const BUTTON_GROUP_LENGTH = 10;
+
+type ButtonConfig = {
+  icon: TemplateResult<1>;
+  tooltip: string;
+  childFlavour: NoteChildrenFlavour;
+  childType: NoteChildrenType;
+};
+
+export const ButtonConfigs: ButtonConfig[] = [
```

Comment:
There are similar configs in https://github.com/toeverything/blocksuite/blob/96046bd805e485886866091140956c6a20551ca2/packages/global/src/config/consts/block-hub.ts

Reply:
I will check. Thanks!